src/main.rs

```diff
@@ -3,12 +3,13 @@ use langer::{ lexer::Lexer, parser::Parser, evaluator::Evaluator };
 use std::fs;
 
 fn main() {
-    let contents = fs::read_to_string("echo.rfeel")
-        .expect("Something went wrong reading the file");
+    let contents = String::from(
+        fs::read_to_string("echo.rfeel")
+        .expect("Something went wrong reading the file"));
 
     println!("With text:\n{}", contents);
 
-    let mut l = Lexer::new(String::from(contents));
+    let mut l = Lexer::new(contents);
```

Comment:
That is OK. I see my old code has an issue too. Would be better if the function didn't pass ownership and passed a reference. I am doing that in a bigger project